worker/jobTypes/fastlyJob.js

```diff
@@ -0,0 +1,54 @@
+const mongo = require('../utils/mongo');
+
+class FastlyJobClass {
+    // pass in a job payload to setup class
+    constructor(currentJob) {
+        this.currentJob = currentJob;
+    }
+
+    // connects to dochub database and upsert {source: target} mappings
+    // to the fastly edge dictionary
+    connectAndUpsert(MongoClient, fastly) {
+        MongoClient.connect(mongo.url, function(err, client) {
```

Comment:
We could put the MongoClient connect stuff in the mongo utils class directly. In the event we decide to change the backend db...as odd as that would be...we could just swap out the implementation in the utility.

Reply:
Would it be better to instead add database and collection parameters to `connectAndUpsert()`? `mongo.js` seems to make use of environment variables in most its functions. I suppose we could also add environment variables for the `dochub` database and its `keys` collection, but that may make the `docker run` call too bulky.

Reply:
Sorry I missed this! If you have time on Monday...I'd rather pull all of the mongodb dependencies out of the Fastly component. The reason is that since we are following an adapter design strategy, having two integration points bound to each in one class breaks the abstraction. It's easier to maintain distributed applications if their bindings are limited in scope. Gives us better plugability.

Reply:
Ah, makes sense!